frontend/src/pages/ResourcesDashboard/index.scss

```diff
@@ -0,0 +1,24 @@
+@use '../../colors.scss' as *;
+
+.grid-container {
+    max-width: 100%;
+}
+
+@media(max-width: 768px) {
+    .ttahub-resources-dashboard--filters {
+        align-items: flex-start;
+        flex-direction: column;
+    }
+}
+
+.ttahub--resource-dashboard-title {
```

Comment:
it definitely seems like we could move this to a "PageTitle" or even a "DashboardTitle" component and keep the css in one place rather than having it duped.